client/src/components/History/History.jsx

```diff
@@ -113,7 +120,39 @@ export const Routes = () => {
                     type: 'GET_JOB_DETAILS',
                     payload: { jobUuid },
                   });
-                  return <JobHistoryModal uuid={jobUuid} />;
+                  return <JobHistoryModal uuid={jobUuid} version="v3" />;
+                }}
+              />
+            </>
+          );
+        }}
+      />
+
+      <Route
+        path={`${root}${ROUTES.JOBSV2}`}
+        render={({ location: { pathname, state } }) => {
+          const locationState = state || {};
+          // Only mark as read if in pure job history view
+          if (
+            pathname === `${root}${ROUTES.JOBSV2}` &&
+            !locationState.fromJobHistoryModal
+          ) {
+            dispatch({
+              type: 'GET_V2_JOBS',
+              params: { offset: 0 },
+            });
+          }
+          return (
+            <>
+              <JobHistory className={styles.content} />
+              <Route
+                path={`${ROUTES.WORKBENCH}${ROUTES.HISTORY}${ROUTES.JOBSV2}/:jobId`}
+                render={({
+                  match: {
+                    params: { jobId },
+                  },
+                }) => {
+                  return <JobHistoryModal uuid={jobId} version="v2" />;
                 }}
```

Comment:
related to other comments, we'll need to consider if we want to show Job Status and Output Location for v2 jobs

![Screenshot 2023-02-10 at 2 03 21 PM](https://user-images.githubusercontent.com/8287580/218186999-4b18b4b2-49a7-45f8-b2f1-0f8a85922335.png)

Reply:
I've updated the data from the rows output location should be null on v2, and I set job status to be "Archived".